src/main/java/io/leonis/subra/game/data/Referee.java

```diff
@@ -35,6 +36,16 @@
    */
   double getCommandTimeStamp();
 
+  /**
+   * @return the {@link TeamIdentity} on the {@link FieldHalf#POSITIVE}.
```

Comment:
should start with proper capitals. doesn't it also read a bit weird as `the TeamIdentity on the FieldHalf#POSITIVE`? or whatever text it renders to